package.json

```diff
@@ -4,7 +4,9 @@
     "license": "MIT",
     "scripts": {
         "version": "node -p \"require('./package.json').version\"",
-        "changelog": "conventional-changelog -i CHANGELOG.md -s -r 0 && git add CHANGELOG.md"
+        "changelog": "conventional-changelog -i CHANGELOG.md -s -r 0 && git add CHANGELOG.md",
```

Comment:
What does this line do? does it generate and commit changelogs automatically?

Reply:
This line updates a changelog file and adds it to version control as far as i know,
was here before me i added the ", "😄